### ISSUE TITLE
Numo::DFloat#max_index

### ISSUE THREAD

**Issue by hatappi** [ruby-numo/narray#66]
When I use `Numo::DFloat`, `Numo::Int32` instance `max_index` method, I found difference.
gem version is `0.9.0.7`.

Numo::Int32#max_index return value is correct.
But Numo::DFloat#max_index return value is incorrect.

## Numo::Int32

```rb
> x = Numo::Int32[[1, 2, 3], [4, 5, 6]]
# => Numo::Int32#shape=[2,3]
# [[1, 2, 3],
#  [4, 5, 6]]
> x.max_index
# => 5
> x.max_index(axis=0)
# => Numo::Int32#shape=[3]
# [3, 4, 5]
> x.max_index(axis=1)
# => Numo::Int32#shape=[2]
# [2, 5]
```

## Numo::DFloat

```rb
> x = Numo::DFloat[[1, 2, 3], [4, 5, 6]]
# => Numo::DFloat#shape=[2,3]
# [[1, 2, 3],
#  [4, 5, 6]]
> x.max_index
# => 4
> x.max_index(axis=0)
# => Numo::Int32#shape=[3]
# [0, 1, 2]
> x.max_index(axis=1)
# => Numo::Int32#shape=[2]
# [2, 4]
```

**Comment by masa16** [comment]
This bug is fixed in https://github.com/ruby-numo/narray/commit/abe5ea8aa92a18ea7c7eaec8e46e8b7738426ec4
I released new version 0.9.0.8 now.
Thanks

**Comment by hatappi** [comment]
I confirmed.
Thank you bug fix.

```rb
[1] pry(main)> Numo::NArray::VERSION
=> "0.9.0.8"
[2] pry(main)> y = Numo::DFloat[[0.1, 0.7, 0.2],[8.0, 1.0, 2.0],[-8.0, 1.0, 2.0],[-8.0, -1.0, -2.0]]
=> Numo::DFloat#shape=[4,3]
[[0.1, 0.7, 0.2],
 [8, 1, 2],
 [-8, 1, 2],
 [-8, -1, -2]]
[3] pry(main)> y.max_index(axis=1)
=> Numo::Int32#shape=[4]
[1, 3, 8, 10]
```
